inc/nav-main.php

```diff
@@ -19,7 +19,7 @@
 </div><!-- end hamburger icon -->
 <nav id="nav-main" class="nav-main flex-container">
   <div class="link-primary flex-container flex-end">
-    <a href="/search" class="no-underline">Search</a>
+    <a href="/search" class="no-underline search-link">Search</a>
```

Comment:
Fix for the "Search" menu item in the main nav going out of it's box in some browsers.